### DISCUSSION TITLE
使用milvus-backup备份服务时,milvus数据缓存问题?

### DISCUSSION THREAD

**Discussion by Ctlcc** [milvus-io/milvus#30269] [Q&A and General discussion]
在尝试使用milvus-backup第三方备份服务时，我遇到了一些问题。首先，我用zilliz/attu删除了一些数据，然后在大约10秒后刷新了zilliz/attu的页面，但是发现数据的实体数并没有变化，还是和删除前一样。更糟糕的是，当我试图使用milvus-backup新建备份API时，之前删除的数据竟然又被备份到了原有的collection中。

这个问题发生在以下环境中：
milvus-backup版本：V0.4.5
milvus版本：V2.3.0
部署环境：Rocky Linux 9.2
我正在寻求解决这个问题的方法，希望能够得到一些有用的建议。

**Comment by yhmo** [comment]
Approx Count 是粗略的统计，不计算被删除的行数
你在这个页面里可以看到精确的统计行数  Entities Count：
![Screenshot from 2024-01-25 15-53-38](https://github.com/milvus-io/milvus/assets/2282099/8144d090-9823-4bed-875f-1efa3dd807af)


如果你的表近期(10分钟以内)有数据的改动，那么在使用milvus-backup之前调用一下collection.flush()以使得所有近期的操作都落盘。只有落盘的数据才会milvus-backup看到。
